pylint_google_style_guide_imports_enforcing/__init__.py

```diff
@@ -21,27 +21,30 @@ class AlphabeticallySortedImports(BaseChecker):
 
     priority = -1
 
+    excluded_modules = ['typing', 'typing_extensions', 'six.moves']
+
     def visit_importfrom(self, node):
         imported = node.do_import_module()
-        for name, _ in node.names:
-            _, result = imported.lookup(name)
-            if not result:
-                # it's not there, so probably this is another module - fine.
-                continue
-            imported_node, *_ = result
-            if isinstance(imported_node, scoped_nodes.Module):
-                continue
-
-            # maybe a submodule?
-            try:
-                imported.import_module(name, relative_only=True)
-            except astroid.AstroidImportError:
-                self.add_message(
-                    self.ONLY_IMPORTING_MODULES_IS_ALLOWED,
-                    node=node,
-                    args=(name, )
-                )
+
+        if imported.name not in self.excluded_modules:
```

Comment:
[minor] Short-circuit is preferrable, i.e.:
```
if imported.name in self.excluded_modules:
    return

...(existing logic)
```
This reduces indents and also makes the diff much shorter

Reply:
That's accurate. Changed :+1: